### PR TITLE
add login information in README.md

### PR DESCRIPTION
**Pull request by puneet-pdx**
Related to issue: #

### Description:

The utility network micro app crashes if credentials are not specified.

### Summary of changes:

- Add info in Readme to add username and password
- Modify micro app to not crash when username and password are absent
- show the error on screen instead of circular progress indicator

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/219/
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

File: microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt [hunk 1]
```diff
@@ -90,8 +100,15 @@ fun UtilityNetworkTraceApp(traceViewModel: TraceViewModel) {
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.Center
         ) {
-            CircularProgressIndicator(
-                modifier = Modifier.size(32.dp)
+            Icon(
```
Comment (shubham7109) [inline microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt:103]: The `Column` on line `98` needs to be wrapped in a Scaffold/Surface in order for the theme to be applied when displaying the below text & icon.

Comment (puneet-pdx) [reply microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt:103]: This is in the micro app, do we really need to worry about applying the theme here?

Comment (shubham7109) [reply microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt:103]: @puneet-pdx I would strongly suggest that we should. All microapps use the `:microapps-lib` module to apply a common theme.

Comment (puneet-pdx) [reply microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt]: Updated

File: microapps/UtilityNetworkTraceApp/README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 # UtilityNetworkTraceApp
 
 This micro app showcases the use of the UtilityNetworks Trace toolkit component. It uses a webmap that comprises of the Utility Network data of naperville utilities.
+The webmap is secured and requires username/password to access the data. Please add the username and password in the `local.properties` file as shown below.
 
+```text
```
Comment (shubham7109) [inline microapps/UtilityNetworkTraceApp/README.md:6]: Small change to get the syntax highlighting
```suggestion
```properties
```

File: microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt [hunk 3]
```diff
@@ -64,35 +71,62 @@ class MainActivity : ComponentActivity() {
...
+                    MainScreen(traceViewModel)
+                }
+                isInitializationError -> {
+                    Column(
+                        modifier = Modifier.fillMaxSize(),
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                        verticalArrangement = Arrangement.Center
+                    ) {
+                        Icon(
+                            Icons.Default.Info,
```
Comment (shubham7109) [inline microapps/UtilityNetworkTraceApp/app/src/main/java/com/arcgismaps/toolkit/utilitynetworktraceapp/MainActivity.kt:107]:
```suggestion
                            imageVector = Icons.Default.Info,
```

**Review by shubham7109 — APPROVED** [review]
@puneet-pdx, LGTM 👍 

Added some minor comments below

**Pull request merged**